tox.ini

```diff
@@ -261,8 +261,6 @@ commands =
 addopts = -ra -q --color=yes
 norecursedirs = .* *.egg* config docs dist build
 xfail_strict = True
-rsyncdirs = src tests
```

Comment:
What were these for?

Reply:
I don't know, but they trigger deprecation warnings in pytest and the messages imply the options were removed

Reply:
Seems to be a long deprecated argument of pytest-xdist which we are not even using. :thinking: Alright, then.